book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] as part of the requirements the rentals are `Getters and Setters` so kindly add them in attr_accessor 🙏